database_cleanup/model/purge_modules.py

```diff
@@ -106,13 +106,18 @@ def default_get(self, cr, uid, fields, context=None):
 
     def find(self, cr, uid, context=None):
         module_pool = self.pool['ir.module.module']
+        purge_line_pool = self.pool['cleanup.purge.line.module']
         module_ids = module_pool.search(cr, uid, [], context=context)
         res = []
         for module in module_pool.browse(cr, uid, module_ids, context=context):
             if get_module_path(module.name):
                 continue
             if module.state == 'uninstalled':
-                module_pool.unlink(cr, uid, module.id, context=context)
+                purge_line_pool.purge(
+                    cr, uid, [
+                        purge_line_pool.create(
+                            cr, uid, {'name': module.name}, context=context),
```

Comment:
It's a style nitpick, but IMO the could would be more readable if the `create` was done in it's own instruction.